### PR TITLE
fixed screenshots bug

### PR DESCRIPTION
**Pull request by MohamedBechirMejri**
# Description

can't take screenshots on the new reddit design. this is a temporary fix for it thanks to @SpaYco 

# Issue Fixes

#1647 

None

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [ ] I am using the recommended development environment
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have formatted and linted my code using python-black and pylint
- [ ] I have cleaned up unnecessary files
- [ ] My changes generate no new warnings
- [ ] My changes follow the existing code-style
- [ ] My changes are relevant to the project

# Any other information (e.g how to test the changes)

None

### CODE REVIEW

File: video_creation/screenshot_downloader.py [hunk 1]
```diff
@@ -214,9 +212,12 @@ def get_screenshots_of_reddit_posts(reddit_object: dict, screenshot_num: int):
...
                 try:
-                    page.locator(f"#t1_{comment['comment_id']}").screenshot(
-                        path=f"assets/temp/{reddit_id}/png/comment_{idx}.png"
-                    )
+                    permalink_arr = comment["comment_url"].split("/")
+                    permalink_arr[-3] = "comment"
+                    permalink = "/".join(permalink_arr)
+                    page.locator(
+                        f"shreddit-comment[permalink='{permalink}']"
+                    ).screenshot(path=f"assets/temp/{reddit_id}/png/comment_{idx}.png")
```
Comment (electro199) [inline video_creation/screenshot_downloader.py:220]: It will only work in non logged in state

**Review by electro199 — CHANGES_REQUESTED** [review]
need changes

**Comment by liamb13** [comment]
I've also attempted to fix the issue here: https://github.com/elebumm/RedditVideoMakerBot/pull/1663

I personally think the redesign looks better for the videos, so my approach removes the opt-out cookie

**Comment by OpenSourceSimon** [comment]
#1663 is a better fix

**Pull request closed without merging**